website/docs/docs/build/incremental-microbatch.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "About microbatch incremental models"
+description: "Learn about the 'microbatch' strategy for incremental models."
+id: "incremental-microbatch"
+---
+
+# About microbatch incremental models <Lifecycle status="beta" />
+
+:::info Microbatch
```

Comment:
@jtcohen6 Thoughts on calling out that this _feature_ is in beta - as I anticipate there will be some changes ahead of our 1.9 final release based on community feedback

Reply:
added beta in there as Jerco originally added it and i removed it since i added in the h1! sorry!